src/main/java/com/example/pfairplayservice/controller/MemberController.java

```diff
@@ -36,12 +43,9 @@
 
     @PostMapping("/member")
     public ResponseEntity<Void> createMember(@RequestBody Member saveMember) {
-        if (saveMember.getId().length() > 10) throw new LengthOverException("id는 10자를 초과 할 수 없습니다.");
-        if (saveMember.getName() == null) throw new RequiredParamNotFoundException("이름을 정확히 입력해주세요");
-        if (saveMember.getBirthday() == null) throw new RequiredParamNotFoundException("생년월일을 정확히 입력해주세요");
-        if (saveMember.getAddress() == null) throw new RequiredParamNotFoundException("주소를 정확히 입력해주세요");
-        if (saveMember.getPhoneNumber() == null) throw new RequiredParamNotFoundException("이름을 정확히 입력해주세요");
-
+        EntityExceptionHandler.MemberPostExceptionHandler(saveMember);
+        Optional<MemberEntity> member = memberRepository.findByMemberId(saveMember.getId());
+        if (member.isPresent()) return ResponseEntity.status(HttpStatus.CONFLICT).build();
         memberRepository.save(saveMember.toMemberEntity());
         return ResponseEntity.status(HttpStatus.CREATED).build();
```

Comment:
Need indentation.
Use auto-indent in IDE.

Reply:
I will